src/store/modules/DownloadModule.ts

```diff
@@ -93,24 +93,19 @@ export const DownloadModule = {
     },
 
     mutations: {
-        updateDownload(state: State, update: UpdateObject) {
-            const newDownloads = [...state.allDownloads];
-            const index = newDownloads.findIndex((old: DownloadProgress) => {
-                return old.assignId === update.assignId;
-            });
-
-            if (index === -1) {
-                // The DownloadProgress by the ID from the update wasn't found at all.
-                console.warn(`Couldn\'t find DownloadProgress object with assignId ${update.assignId}.`);
-                return;
+        remove(state: State, assignId: number) {
```

Comment:
Minor: we have `addDownload` and `updateDownload` so this should probably be `removeDownload`.

Nitpick: to match `updateDownload` this could perhaps take `UpdateObject` as the argument instead of the `assignId`.